tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/NetStandardLibraries/Elasticsearch/ElasticsearchElasticClient.cs

```diff
@@ -204,9 +198,7 @@ public override void GenerateError()
 
             var client = new ElasticsearchClient(settings);
 
-#pragma warning disable CS0618 // obsolete usage is ok here
-            var response = client.Ping();
-#pragma warning restore CS0618
+            var response = _ = await client.PingAsync();
```

Comment:
Kind of odd syntax, not sure you need the discard here.

Reply:
Ooops, not sure how that happened.  I'll fix it.